src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.ConflictResolution.targets

```diff
@@ -25,6 +25,20 @@ Copyright (c) .NET Foundation. All rights reserved.
           To do this, we look at the files that would be copied local when CopyLocalLockFileAssemblies is true
           -->
       <_LockFileAssemblies Include="@(AllCopyLocalItems->WithMetadataValue('Type', 'assembly'))" />
+
+      <!--RuntimeTarget-->
+      <_RuntimeTargetItems Include="@(_ActiveTFMFileDependencies->WithMetadataValue('FileGroup', 'RuntimeTarget'))" />
```

Comment:
Just sdk is fine.  I think we should probably have some naming convention for private items to help avoid conflicts but it doesn't make sense to start establishing that in this change.